### PR TITLE
fixed progress circle issue, started modal for monthly giving cancell…

### PR DESCRIPTION
**Pull request by malawi-mike**
…ation, and make a few other minor tweaks

### CODE REVIEW

File: footer.inc [hunk 1]
```diff
@@ -14,7 +14,7 @@
 					id="download-button"
 					class="btn-large waves-effect waves-light light blue lighten-1 modal-trigger" href='village_fund_payment_view.php' 
 					>Give Monthly</a>	
-				<script>
```
Comment (jdepree) [inline footer.inc]: Can we just delete this section?  Did you replace it below?

File: header.inc [hunk 2]
```diff
@@ -224,7 +225,7 @@ function donateWithStripe(subscribe, amount, title, projectId, firstName, lastNa
 </script>
 
 <!-- form validation code -->
-<script src="https://jqueryvalidation.org/files/lib/jquery.js"></script>
```
Comment (jdepree) [inline header.inc]: Was this doing anything?  Is the line below a direct replacement for it?  Can it be deleted?

Comment (malawi-mike) [reply header.inc]: yes, we can remove it. done

File: footer.inc (hunk 1 above)
Comment (malawi-mike) [reply footer.inc]: yes. i just left it in for you to review and potentially to steal some of the code, if necessary

**Pull request merged**